man/chpl.txt

```diff
@@ -114,7 +112,7 @@ OPTIONS
   --[no-]ieee-float   Disable [enable] optimizations that may affect IEEE
                     floating point conformance. The default is whatever level
                     of optimization/IEEE floating point support your C compiler
-                    provides with the C compiler optimization provided.
+                    provides at a given optimization level.
```

Comment:
"provides at a given..." still seems vague (for me it begs the question "which one?").  I suggested "the given" as a means of suggesting that someone was specifying it without going to the trouble of saying who...  (the answer effectively being "typically 'chpl' as steered by the user's choice of other options").  We could be slightly more precise by saying "provides for the optimization level specified by 'chpl'".  

(To be fully precise, I think we'd need to refer to the -O option, the -cflags option, environment variables, etc. but that feels like overkill to me.)